Bottom: 2813cf84408c3d52a2af259914f650f0d21e5d90
Top:    8f9b019646fc91c98655b6eaf77d32ed5ead9a98
Author: student <student@ReliableEmbeddedSystems.com>
Date:   2020-08-26 11:15:29 +0300

make bsp tool dunfell compatible

Signed-off-by: student <student@ReliableEmbeddedSystems.com>


---

diff --git a/scripts/lib/bsp/substrate/target/arch/common/conf/layer.conf b/scripts/lib/bsp/substrate/target/arch/common/conf/layer.conf
index b18c480656..5a914ef1ed 100644
--- a/scripts/lib/bsp/substrate/target/arch/common/conf/layer.conf
+++ b/scripts/lib/bsp/substrate/target/arch/common/conf/layer.conf
@@ -9,4 +9,4 @@ BBFILE_COLLECTIONS += "{{=machine}}"
 BBFILE_PATTERN_{{=machine}} = "^${LAYERDIR}/"
 BBFILE_PRIORITY_{{=machine}} = "6"
 
-LAYERSERIES_COMPAT_{{=machine}} = "sumo thud warrior zeus"
+LAYERSERIES_COMPAT_{{=machine}} = "sumo thud warrior zeus dunfell"
